src/FlowLinesLayer/FlowLinesLayer.ts

```diff
@@ -162,7 +163,7 @@ class FlowLinesLayer extends Layer<Props, LayerState, Context> {
         ],
       );
 
-      const t = 1; // Border thickness
+      const t = this.props.borderThickness || 1; // Border thickness
```

Comment:
Let's put the default value to `defaultProps` to be consistent with the other props.